src/Apps/Sell/Components/ImagePreviewItem.tsx

```diff
@@ -119,7 +113,13 @@ export const ImagePreviewItem: React.FC<ImagePreviewItemProps> = ({
           />
         )}
 
-        {photo.loading && photo.progress && (
+        {!!isProcessing && !isLoading && (
```

Comment:
When loading the image fails or Gemini hasn't processed the image yet, we show the "processing..." message instead of the spinner. 

The spinner has worked well, but can be confusing when the user reloads the page before Gemini has processed the images. This results in a spinner that never stops because there is no reloading logic.